src/components/ApeInfoTooltip/ApeInfoTooltip.tsx

```diff
@@ -45,10 +49,14 @@ export const ApeInfoTooltip = ({
         tooltip: clsx(localClasses.tooltip, classes?.tooltip),
       }}
       interactive
+      leaveDelay={100} // delay is important to make sure user can click link in tooltip
```

Comment:
interactive should make that unnecessary?

Reply:
See my earlier review comment

Reply:
oh okay, yeah I guess the mouseover isn't contiguous, it had worked for ProfileCard